Nasty patch for consolehelper. Wireshark run from consolehelper in unprivileged
mode wrongly detects path as /usr/bin instead of /usr/sbin. Causes few strange 
crashes in GUI. 

diff -up wireshark-0.99.7/capture_sync.c.old wireshark-0.99.7/capture_sync.c
--- wireshark-0.99.7/capture_sync.c.old	2007-12-18 02:16:14.000000000 +0100
+++ wireshark-0.99.7/capture_sync.c	2007-12-19 13:50:17.000000000 +0100
@@ -230,7 +230,7 @@ init_pipe_args(int *argc) {
     *argv = NULL;
 
     /* take Wireshark's absolute program path and replace "Wireshark" with "dumpcap" */
-    exename = g_strdup_printf("%s" G_DIR_SEPARATOR_S "dumpcap", progfile_dir);
+    exename = g_strdup_printf("%s" G_DIR_SEPARATOR_S "dumpcap", "/usr/sbin");
 
     /* Make that the first argument in the argument list (argv[0]). */
     argv = sync_pipe_add_arg(argv, argc, exename);
